src/rust/metric-forwarder/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "metric-forwarder"
+version = "0.0.1"
+authors = ["Max Wittek <wimax@graplsecurity.com>"]
+edition = "2018"
+
+[dependencies]
+grapl-config = {path="../grapl-config", version="*"}
+
+async-trait = "0.1.22"
+aws_lambda_events = "0.2.1"
+base64 = "0.10.1"
+flate2 = "1.0.6"
+lambda_runtime = "0.2.*"
+log = "0.4.6"
+rusoto_cloudwatch = "0.45.0"
+serde_json = "1.0.53"
+thiserror = "1.0.20"
+tokio-compat = "0.1.*"
+tokio = { version = "0.2.*", features = ["sync", "rt-core", "macros", "time", "rt-threaded"] }
```

Comment:
why tokio? you might ask

well... cuz it's easier to start something off async-friendly than to bolt it on later...

more than happy to revisit this